Gruntfile.js

```diff
@@ -1,164 +1,164 @@
 module.exports = function(grunt) {
-    require("matchdep").filterDev("grunt-*").forEach(grunt.loadNpmTasks);
-
-    var pkg = grunt.file.readJSON('package.json');
-
-    grunt.initConfig({
-        pkg: pkg,
-        shell: {
-
-        },
-
-        sass: {
-            options: {
-                outputStyle: 'nested',
-                imagePath: 'assets/images',
-                precision: 5,
-                includePaths: [
-                    'components'
-                ]
-            },
-            dev: {
-                files: {
-                    'public/css/screen.css': 'assets/sass/config.imports',
-                }
-            }
-        },
-
-        pixrem: {
-            dist: {
-                options: {
-                    replace: false
-                },
-                src: 'public/css/screen.css',
-                dest: 'public/css/screen.css'
-            }
-        },
-
-        uglify: {
-            dev: {
-                options: {
-                    beautify: true,
-                    mangle: false,
-                    compress: false
-                },
-                files: {
-                    'public/js/script.min.js': [
-                        // Include:
-                        'components/jquery/dist/jquery.js',
-                        'assets/js/modules/*.js',
-                        'assets/js/script.js'
-                    ],
-                    'public/js/modernizr.min.js': 'assets/js/modernizr.min.js'
-                }
-            }
-        },
-
-        imagemin: { // Task
-            dynamic: { // Another target
-                files: [{
-                    expand: true, // Enable dynamic expansion
-                    cwd: 'assets/images/', // cwd is 'current working directory' - Src matches are relative to this path
-                    src: ['**/*.{png,jpg,gif}'], // Actual patterns to match
-                    dest: 'public/images/' // Destination path prefix
-                }]
-            }
-        },
-
-        copyFiles: '**/*.{eot,svg,ttf,woff,pdf}',
-        copy: {
-            target: {
-                files: [
-                    // includes files within path
-                    {
-                        expand: true,
-                        cwd: 'assets/',
-                        src: ['<%= copyFiles %>'],
-                        dest: 'public/',
-                        filter: 'isFile'
-                    }
-                ]
-            }
+  require("matchdep").filterDev("grunt-*").forEach(grunt.loadNpmTasks);
+
+  var pkg = grunt.file.readJSON('package.json');
+
+  grunt.initConfig({
+    pkg: pkg,
+    shell: {
+
+    },
+
+    sass: {
+      options: {
+        outputStyle: 'nested',
+        imagePath: 'assets/images',
+        precision: 5,
+        includePaths: [
+          'components'
+        ]
+      },
+      dev: {
+        files: {
+          'public/css/screen.css': 'assets/sass/config.imports.scss'
```

Comment:
This is the only line I changed in this file. Everything else was removing whitespace and fixing the indentation to match the .editorconfig file.